### PR TITLE
Drone: drop MariaDB 10.1 (EOL) add MariaDB 10.5

### PR DESCRIPTION
**Pull request by solracsf**
- Drop MariaDB 10.1 (EOL)
- Add MariaDB 10.5 GA

### CODE REVIEW

**Review by rullzer — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -361,24 +360,30 @@ trigger:
...
+- name: mariadb10.4-php7.4
+  image: nextcloudci/php7.4:php7.4-3
   commands:
     - bash tests/drone-run-php-tests.sh || exit 0
     - NOCOVERAGE=true TEST_SELECTION=DB ./autotest.sh mariadb
+#- name: mariadb10.4-php8.0
+#  image: nextcloudci/php8.0:latest
+#  commands:
+#    - bash tests/drone-run-php-tests.sh || exit 0
+#    - NOCOVERAGE=true TEST_SELECTION=DB ./autotest.sh mariadb
```
Comment (k00ni) [inline .drone.yml:380]: Are these a debug-leftover?

Comment (acsfer) [reply .drone.yml:380]: https://github.com/nextcloud/server/commit/f79f1895d378096506988a8d22ec162b1c8644af

File: .drone.yml [hunk 2]
```diff
@@ -290,14 +290,14 @@ trigger:
 
 ---
 kind: pipeline
-name: mariadb10.1-php7.3
+name: 10.2-php7.3
```
Comment (MorrisJobke) [inline .drone.yml:293]:
```suggestion
name: mariadb10.2-php7.3
```

**Review by MorrisJobke — APPROVED** [review]
Looks good 👍

**Comment by MorrisJobke** [comment]
It seems to not open the port correctly, so it couldn't connect properly.

Bildschirmfoto 2021-05-19 um 11 09 36

**Comment by skjnldsv** [comment]
Hey @acsfer could you rebase ? :)

**Comment by acsfer** [comment]
Close in flavour of https://github.com/nextcloud/server/pull/31011

**Pull request closed without merging**